test/utils/image/manifest.go

```diff
@@ -46,7 +49,41 @@ func (i *ImageConfig) SetVersion(version string) {
 	i.version = version
 }
 
+func initReg() RegistryList {
+	registry := RegistryList{
+		DockerLibraryRegistry: "docker.io/library",
+		E2eRegistry:           "gcr.io/kubernetes-e2e-test-images",
+		GcRegistry:            "k8s.gcr.io",
+		PrivateRegistry:       "gcr.io/k8s-authenticated-test",
+		SampleRegistry:        "gcr.io/google-samples",
+	}
+	repoList := os.Getenv("KUBE_TEST_REPO_LIST")
+	if repoList == "" {
+		return registry
+	}
+
+	fileContent, err := ioutil.ReadFile(repoList)
+	if err != nil {
+		fmt.Print("Error reading '" + repoList + "' file contents.")
+		fmt.Print(err)
```

Comment:
would it be worth panicking on these errors instead? it seems like an error message alone would easily get lost in the noise of the e2e tests.

Reply:
Done